lib/service/services_cli.rb

```diff
@@ -177,6 +176,32 @@ def kill(targets, verbose: false)
       end
     end
 
+    # Restart running services and start stopped ones.
+    def restart(targets, verbose: false)
```

Comment:
The idea of the command classed was that the logic could live there and we could gradually phase out the `services_cli` file. So this should not be moved here.

Reply:
Oh, that wasn't obvious to me. Yeah, I can move that back out. Was I also wrong in assuming that system calls to `launchctl`/`systemctl` should be avoided in command files? It seems like all of those calls are housed in `services_cli.rb` and none of them are in the files associated with individual commands.

Any particular reason why you are planning on phasing out the `services_cli` file? Just curious. Also, I'd be open to working on that kind of stuff in the future.

Reply:
I've avoided them so far in cmd files by just making a wrapper in services_cli. The reason I'm moving away from services_cli is just to avoid the whole application being in a single file that's hard to read and maintain.

Reply:
That makes sense. Cool, I updated it. Let me know if it's what you had in mind.